recipes/pcl/all/test_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+from conans import ConanFile, CMake, tools
+import os
+
+class PclTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+    short_paths = True
```

Comment:
```suggestion
```

short_paths is not added to test_package. If you REALLY need it, pass to the recipe.